docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.rst

```diff
@@ -0,0 +1,108 @@
+        "database.password": "PG_PASSWORD",
+        "database.dbname": "PG_DATABASE_NAME",
+        "database.sslmode": "SSL_MODE",
+        "plugin.name": "wal2json",
+        "slot.name": "PG_SLOT_NAME",
+        "publication.name": "PG_PUBLICATION_NAME",
+        "database.server.name": "KAFKA_TOPIC_PREFIX",
+        "table.include.list": "PG_TABLES",
+        "tasks.max":"NR_TASKS",
+        "key.converter": "io.confluent.connect.avro.AvroConverter",
+        "key.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "key.converter.basic.auth.credentials.source": "USER_INFO",
+        "key.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD",
+        "value.converter": "io.confluent.connect.avro.AvroConverter",
+        "value.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "value.converter.basic.auth.credentials.source": "USER_INFO",
+        "value.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD",
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: the connector name
+* ``PG_HOST``, ``PG_PORT``, ``PG_DATABASE_NAME``, ``SSL_MODE``, ``PG_USER``, ``PG_PASSWORD``, ``PG_TABLES``, ``PG_PUBLICATION_NAME`` and ``PG_SLOT_NAME``: source database parameters collected in the :ref:`prerequisite <connect_debezium_pg_source_prereq>` phase. 
+* ``database.server.name``: the logical name of the database, dictates the prefix that will be used for Apache Kafka topic names. The resulting topic name will be the concatenation of the ``database.server.name`` and the table name.
+* ``tasks.max``: maximum number of tasks to execute in parallel. By default this is 1, the connector can use at most 1 task for each source table defined.
+* ``plugin.name``: defines the `PostgreSQL output plugin <https://debezium.io/documentation/reference/connectors/postgresql.html>`_ to convert changes in the database into events in Apache Kafka.
+* ``key.converter`` and ``value.converter``:  defines the messages data format in the Apache Kafka topic. The ``io.confluent.connect.avro.AvroConverter`` converter pushes messages in Avro format. To store the messages schema we use Aiven's `Karapace schema registry <https://github.com/aiven/karapace>`_ as specified by the ``schema.registry.url`` parameter and related credentials.
+
+.. Note::
+
+    The ``key.converter`` and ``value.converter`` sections are only needed when pushing data in Avro format. If omitted the messages will be defined in JSON format.
+
+
+Create a Kafka Connect connector with Aiven CLI
+'''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, execute the following :ref:`Aiven CLI command <avn_service_connector_create>`, replacing the ``SERVICE_NAME`` with the name of the Aiven service where the connector needs to run:
+
+:: 
+
+    avn service connector create SERVICE_NAME @debezium_source_pg.json
```

Comment:
just silly question so it is like we need to add the `@` on the json file name, correct?

FILE_NAME = debezium_source_pg.json

avn service connector create SERVICE_NAME @FILE_NAME

Reply:
yep you're correct